scripts/benchmark_db/bench.rb

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+# Script to test various configurations that can impact performance of the node
+
+require 'deep_merge'
+require 'fileutils'
+require 'open3'
+require 'optparse'
+require 'pathname'
+require 'set'
+require 'tmpdir'
+require 'toml-rb'
+
+# This is just for capturing the snapshot height
+SNAPSHOT_REGEX = /(?<height>\d+)_.*/.freeze
```

Comment:
So we only support the naming scheme from the manually downloaded Filecoin snapshots? We don't support snapshots that we've downloaded with `forest-cli snapshot fetch`?

Reply:
You mean something like `minimal_finality_stateroots_2382481_2022-11-30_04-00-30.car`?

Reply:
Files downloaded with `forest-cli snapshot fetch` follow this pattern: `{provider}_snapshot_{network}_{date}_height_{height}.car`. Examples: `filecoin_snapshot_mainnet_2022-11-30_height_2382480.car` and `forest_snapshot_calibnet_2022-11-30_height_81393.car`.

Reply:
This naming change was made a few days ago, I think.

Reply:
Right now I don't support `calibnet` at all and scheme like this one too: `forest_snapshot_calibnet_2022-11-30_height_81393.car`

Reply:
We want to support the naming scheme used by `forest-cli snapshot`, right?

Reply:
Ok will fix this!